generators/entity-client/templates/react/src/main/webapp/app/entities/index.tsx.ejs

```diff
@@ -32,13 +32,15 @@ const Routes = ({ match }) => (
   <>
     <Switch>
       <%_ if (!readOnly) { _%>
-      <ErrorBoundaryRoute exact path={`${match.url}/:id/delete`} component={<%= entityReactName %>DeleteDialog} />
       <ErrorBoundaryRoute exact path={`${match.url}/new`} component={<%= entityReactName %>Update} />
       <ErrorBoundaryRoute exact path={`${match.url}/:id/edit`} component={<%= entityReactName %>Update} />
       <%_ } _%>
       <ErrorBoundaryRoute exact path={`${match.url}/:id`} component={<%= entityReactName %>Detail}  />
       <ErrorBoundaryRoute path={match.url} component={<%= entityReactName %>} />
     </Switch>
+    <%_ if (!readOnly) { _%>
+    <ErrorBoundaryRoute exact path={`${match.url}/:id/delete`} component={<%= entityReactName %>DeleteDialog} />
+    <%_ } _%>
```

Comment:
not so sure about that one... if the point of having the deletion modal on top of the entity list page is what matters, then there may be a better way to achieve that

Reply:
Usability wise, I would say this is the correct way to do. The list should be visible beneath the `delete` modal.  So let us do this.